tests/auto/rt_auto.sh

```diff
@@ -8,38 +8,40 @@ else
   exit 1
 fi
 
-export RT_COMPILER='intel'
-source ../detect_machine.sh
-echo "Machine ID: "+$MACHINE_ID
-if [[ $MACHINE_ID = hera.* ]]; then
+if [[ $HOSTNAME == hfe* ]]; then
+  MACHINE_NAME=hera
   WORKDIR=/scratch1/NCEPDEV/nems/Brian.Curtis/test
   export PATH=/scratch1/NCEPDEV/nems/emc.nemspara/soft/miniconda3/bin:$PATH
   export PYTHONPATH=/scratch1/NCEPDEV/nems/emc.nemspara/soft/miniconda3/lib/python3.8/site-packages
-elif [[ $MACHINE_ID = orion.* ]]; then
+elif [[ $HOSTNAME == Orion-login-* ]]; then
+  MACHINE_NAME=orion
   WORKDIR=/work/noaa/nems/bcurtis/test
   export PATH=/work/noaa/nems/emc.nemspara/soft/miniconda3/bin:$PATH
   export PYTHONPATH=/work/noaa/nems/emc.nemspara/soft/miniconda3/lib/python3.8/site-packages
-elif [[ $MACHINE_ID = jet.* ]]; then
+elif [[ $HOSTNAME == fe* ]]; then
+  MACHINE_NAME=jet
   WORKDIR=/lfs4/HFIP/h-nems/Brian.Curtis/test
   export ACCNR="h-nems"
   export PATH=/lfs4/HFIP/hfv3gfs/software/miniconda3/4.8.3/envs/ufs-weather-model/bin:/lfs4/HFIP/hfv3gfs/software/miniconda3/4.8.3/bin:$PATH
   export PYTHONPATH=/lfs4/HFIP/hfv3gfs/software/miniconda3/4.8.3/envs/ufs-weather-model/lib/python3.8/site-packages:/lfs4/HFIP/hfv3gfs/software/miniconda3/4.8.3/lib/python3.8/site-packages
-elif [[ $MACHINE_ID = gaea.* ]]; then
+elif [[ $HOSTNAME == gaea* ]]; then
+  MACHINE_NAME=gaea
   WORKDIR=/lustre/f2/pdata/ncep/Brian.Curtis/test
   export LOADEDMODULES=$LOADEDMODULES
   export ACCNR="nggps_emc" # This applies to Brian.Curtis, may need change later
   export PATH=/lustre/f2/pdata/esrl/gsd/contrib/miniconda3/4.8.3/envs/ufs-weather-model/bin:$PATH
   export PYTHONPATH=/lustre/f2/pdata/esrl/gsd/contrib/miniconda3/4.8.3/lib/python3.8/site-packages
-elif [[ $MACHINE_ID = cheyenne.* ]]; then
-  #export PATH=/glade/p/ral/jntp/tools/ecFlow-5.3.1/bin:$PATH
-  #export PYTHONPATH=/glade/p/ral/jntp/tools/ecFlow-5.3.1/lib/python2.7/site-packages
-  echo "cheyenne not currently supported. automated RT not starting"
-  exit 1
+elif [[ $HOSTNAME = cheyenne*.cheyenne.ucar.edu ]]; then
```

Comment:
This needs to be just `cheyenne*`, I think, because:
```
+ [[ cheyenne1 = cheyenne*.cheyenne.ucar.edu ]]
+ echo 'No Python Path for this machine. automated RT not starting'
```

Reply:
Will do, i don't have cheyenne access so I guesses based on what other files had in them.

Reply:
Done